### PR TITLE
Remove unsafe inline in CSP policy and update svg fill

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
This removes unsafe-inline from CSP policy, removes inline CSS "fill" from our SVG components and adds svg native fill property to these. One unit test had to be amended. Some of our SVGs (Logos) use inline styles for colour fills, however these do not work without explicitly allowing the unsafe-inline CSP policy which is considered unsafe. We needed to get the SVGs updated to not use CSS.

### How to review
Visually check SVG logos in different "test_theme_" schemas, check if svg components are looking the same after the change.

### Checklist
* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by VirajP1002** [comment]
I noticed one of the logos looked slightly different. Does something need to be changed? Not sure if it's on my end.

Main: 
image

Your branch:
image

**Comment by petechd** [comment]
> I noticed one of the logos looked slightly different. Does something need to be changed? Not sure if it's on my end.
> 
> Main: image
> 
> Your branch: image

Fixed now, I didn't get rid off all the CSS properties in some SVGs, well spotted and thanks!

**Comment by petechd** [comment]
> I noticed one of the logos looked slightly different. Does something need to be changed? Not sure if it's on my end.
> 
> Main: image
> 
> Your branch: image

Well spotted, there were some remaining CSS fill properties in that file, removed now, thanks.

**Review by liamtoozer — APPROVED** [review]
SVGs are displayed great for me 👍

**Comment by VirajP1002** [comment]
All but one looks good now! Just one alignment issue on test_theme_ukhsa_ons:

Main:
Screenshot 2024-04-10 at 11 39 51

Your branch:
Screenshot 2024-04-10 at 11 40 05

Not too sure if it's just for me

File: templates/assets/images/ons-logo-stacked.svg [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<svg class="ons-icon--logo ons-u-mb-xs" style="display:inline;" xmlns="http://www.w3.org/2000/svg" width="120" height="27" viewBox="0 5 595 116" aria-labelledby="ons-logo-stacked-en-alt">
+<svg class="ons-u-di ons-icon--logo ons-u-mb-xs" xmlns="http://www.w3.org/2000/svg" width="120" height="27" viewBox="0 5 595 116" aria-labelledby="ons-logo-stacked-en-alt">
```
Comment (rmccar) [inline templates/assets/images/ons-logo-stacked.svg:1]: I got the same issue as Viraj, I think this fixes the issue

```suggestion
<svg class="ons-u-di ons-u-mb-xs" xmlns="http://www.w3.org/2000/svg" width="120" height="27" viewBox="0 5 595 116" aria-labelledby="ons-logo-stacked-en-alt">
```

**Comment by liamtoozer** [comment]
Quick follow-up question [from this](https://github.com/ONSdigital/eq-questionnaire-runner/pull/1349#discussion_r1559580243) for @VirajP1002 & @rmccar - were you testing this locally? I tested locally but the images seemed to be aligned for that schema. Were you testing in a staging/test env instead? I just wanted to double-check what I should be looking for next time 👀

**Comment by rmccar** [comment]
> Quick follow-up question [from this](https://github.com/ONSdigital/eq-questionnaire-runner/pull/1349#discussion_r1559580243) for @VirajP1002 & @rmccar - were you testing this locally? I tested locally but the images seemed to be aligned for that schema. Were you testing in a staging/test env instead? I just wanted to double-check what I should be looking for next time 👀

I was just testing locally, try checking out the commit before the last one and launching the test_theme_ukhsa_ons schema. Thats all I did, maybe you missed it? Then check out the latest one and they should be aligned

**Comment by liamtoozer** [comment]
> > Quick follow-up question [from this](https://github.com/ONSdigital/eq-questionnaire-runner/pull/1349#discussion_r1559580243) for @VirajP1002 & @rmccar - were you testing this locally? I tested locally but the images seemed to be aligned for that schema. Were you testing in a staging/test env instead? I just wanted to double-check what I should be looking for next time 👀
> 
> I was just testing locally, try checking out the commit before the last one and launching the test_theme_ukhsa_ons schema. Thats all I did, maybe you missed it? Then check out the latest one and they should be aligned

Ah, ignore me, I just double-checked in staging - I didn't realise that the bottom SVG always needs to look indented from the top SVG. I think I saw them both aligned locally and thought that was the expected layout. Thanks for your help 👍

**Comment by rmccar** [comment]
> > > Quick follow-up question [from this](https://github.com/ONSdigital/eq-questionnaire-runner/pull/1349#discussion_r1559580243) for @VirajP1002 & @rmccar - were you testing this locally? I tested locally but the images seemed to be aligned for that schema. Were you testing in a staging/test env instead? I just wanted to double-check what I should be looking for next time 👀
> > 
> > 
> > I was just testing locally, try checking out the commit before the last one and launching the test_theme_ukhsa_ons schema. Thats all I did, maybe you missed it? Then check out the latest one and they should be aligned
> 
> Ah, ignore me, I just double-checked in staging - I didn't realise that the bottom SVG always needs to look indented from the top SVG. I think I saw them both aligned locally and thought that was the expected layout. Thanks for your help 👍

Nah its wrong in main too, they're both supposed to be next to each other but yeah sounds like you're seeing the right thing. If you check this branch out now you should see them how they're supposed to be

**Review by rmccar — APPROVED** [review]

**Review by rmccar — APPROVED** [review]

**Comment by liamtoozer** [comment]
> > > > Quick follow-up question [from this](https://github.com/ONSdigital/eq-questionnaire-runner/pull/1349#discussion_r1559580243) for @VirajP1002 & @rmccar - were you testing this locally? I tested locally but the images seemed to be aligned for that schema. Were you testing in a staging/test env instead? I just wanted to double-check what I should be looking for next time 👀
> > > 
> > > 
> > > I was just testing locally, try checking out the commit before the last one and launching the test_theme_ukhsa_ons schema. Thats all I did, maybe you missed it? Then check out the latest one and they should be aligned
> > 
> > 
> > Ah, ignore me, I just double-checked in staging - I didn't realise that the bottom SVG always needs to look indented from the top SVG. I think I saw them both aligned locally and thought that was the expected layout. Thanks for your help 👍
> 
> Nah its wrong in main too, they're both supposed to be next to each other but yeah sounds like you're seeing the right thing. If you check this branch out now you should see them how they're supposed to be

Yep looks all good now, thanks 👍

**Comment by petechd** [comment]
I've also tested this in my staging and looking good so will merge it now.

**Pull request merged**